Process_of_Authoring_GitHub_Actions_for_Azure.md

```diff
@@ -0,0 +1,98 @@
+
+## Steps to create a new Action repo 
+
+If you are a Microsoft employee, 
+1. Request a new repo from the below links as suitable:
+  - [https://repos.opensource.microsoft.com/orgs/Azure/new](https://repos.opensource.microsoft.com/Azure) for creating repos under github.com/Azure Org
+  - [https://repos.opensource.microsoft.com/orgs/Microsoft/new](https://repos.opensource.microsoft.com/microsoft) for creating repos under github.com/Microsoft Org
+
+Ensure to select **Engineering system: GitHub for Open Source** as GitHub Actions for Azure or Microsoft are published as Open Source in repos created in
+Microsoft's official GitHub.com organizations, powered by GitHub Enterprise Cloud.
+
+2. Alternatively, Microsoft users who might already have required permissions on github.com/azure and github.com/Microsoft orgs could just directly create a repo in the respective org without going through the above open-source portal URLs. 
+
+Either-ways, there is an Open Source compliance request that automatically gets created at the end of the repo creation step. Creators receive an e-mail to complete the additional steps required to complete the setup process.
+
+If you are an external (non-Microsoft) contributor, please reach out to the [ACE Team](mailto:ace-team@github.com) for help. 
+
+## Guidelines for setting permissions on the repo
+Once the repo is created, following is the permission model that is followed across various action repos within the [**Azure**](https://github.com/Azure) and [**Microsoft**](https://github.com/Microsoft) GitHub orgs:
+
+- [actions-admin](https://github.com/orgs/Azure/teams/actions-admin/members) team would be added with "admin" role to every new GH repo created for authoring any new action under the Azure org (github.com/azure/). This will ensure that ACE team can act in admin role to ensure that the process guidelines for authoring & publishing the actions are completely followed.
+
+- Owners of the individual actions could manage the "maintainer" permissions for their teams by [creating a new github team handle](https://github.com/orgs/Azure/new-team) for their crews and adding this handle with "admin"/"maintainer" role as needed. 
+
+**Note:** It is recommended that team handles are used for maintaining permissions instead of adding individual member's github handles.
+
+- Kindly request the corresponding Azure product teams to include their team handle as "Admins"/"Maintainers" to facilitate raising any commits from their side for feature enhancements or for fixing supportability issues
+
+
+## ReadMe.md &amp; action.yml files
+
+Once the Action development is complete, ensure that it is documented well in the **readme.md** file at the root of the repo of the Master branch. Please note that this content is used when the Action is published to the GitHub Marketplace. Hence including a sample workflow which covers the best usage of the Action is always recommended.
```

Comment:
@kaverma - This is more check (presence of Readme) that can be done by repo readiness tool. I see few more such candidates below.

Reply:
Yep makes sense, added in the corresponding feature too.

Reply:
https://github.com/github/GitHub-EcoSystem/issues/1075